app/src/api/multishot.lisp

```diff
@@ -48,6 +48,18 @@
            addresses)
   results)
 
+(defun merge-results (result-tables)
```

Comment:
https://github.com/rigetti/qvm/blob/3a2cc06ff4b041ef8e6e350f06cf480514882231/app/src/api/multishot.lisp#L51-L58
Is there no library function for merging hash tables? I checked quickly online and did not find one.

Reply:
@jlapeyre There isn't one. It's not clear what the merge policy would even be as a default.